task/buildah/0.1/buildah.yaml

```diff
@@ -67,6 +67,10 @@ spec:
     description: Target stage in Dockerfile to build. If not specified, the Dockerfile is processed entirely to (and including) its last stage.
     type: string
     default: ""
+  - name: ENTITLEMENT_SECRET
```

Comment:
the changes is for adding support to download rpms during builds. Do we need to add a e2e-tests to cover this scenario?

Reply:
We do, part of the acceptance criteria is "Create a follow up story for e2e tests and link it with this story".

Reply:
oh I see.. thanks for pointing it now. changes looks good to me.